src/Bynder/Api/Impl/Upload/FileUploader.php

```diff
@@ -24,47 +25,26 @@ class FileUploader
      */
     const CHUNK_SIZE = 1024 * 1024 * 5;
 
-    /**
-     * Max polling iterations to wait for the asset to be converted.
-     */
-    const MAX_POLLING_ITERATIONS = 60;
-
-    /**
-     * Idle time between iterations in milliseconds.
-     */
-    const POLLING_IDLE_TIME = 2000;
-
-    /**
-     * Max polling iterations to wait for the asset to be converted.
-     */
-    const MAX_CONCURRENT_CHUNKS = 1;
-
     /**
      *
      * @var AbstractRequestHandler Request handler used to communicate with the API.
      */
     private $requestHandler;
 
     /**
-     * @var AmazonApi Amazon API used to upload parts.
-     */
-    private $amazonApi;
-
-    /**
-     * AWS bucket Url to upload chunks.
+     *
+     * @var string sha256 digest of the file to be uploaded.
      */
-    private $awsBucket;
+    private $fileSha256;
```

Comment:
Is there a particular reason the checksum is put in an instance variable whereas the other variables are simply passed to the other methods? It feels a bit inconsistent.